specification/synapse/resource-manager/Microsoft.Synapse/preview/2021-06-01-preview/sqlServer.json

```diff
@@ -1526,7 +1526,12 @@
         },
         "isStorageSecondaryKeyInUse": {
           "description": "Specifies whether storageAccountAccessKey value is the storage's secondary key.",
-          "type": "boolean"
+          "type": "boolean",
+          "x-ms-mutability": [
```

Comment:
You dont need to specify x-ms-mutability with all 3 values as thats the default value

Reply:
Suggestion in S360 is recommended to add x-ms-mutability with create, read. https://portal.azure-devex-tools.com/amekpis/correctness/detail?errorId=02E7242E-FCF8-41A8-B7CF-0D3CDF06E5B0 Do I keep just create, read and remove update ?